proxy/common/types.py

```diff
@@ -8,6 +8,7 @@
     :copyright: (c) 2013-present by Abhinav Singh and contributors.
     :license: BSD, see LICENSE for more details.
 """
+
```

Comment:
This is clearly unrelated to changing the if-clause. It's strongly recommended not to mix style changes with the functional ones. Besides, it's best to ask the maintainer if they even want those in the first place. Also, looking at other modules, they actually consistently don't have this extra empty line here.
```suggestion
```

Reply:
@yezz123 Once #658 is in, we should be able to get this in.  Regarding extra line, I think it's not needed.

You'll also need to update your PR as currently `This branch is out-of-date with the base branch`.

Thank you